airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -70,7 +69,6 @@ def send_request(self, query: str, customer_id: str) -> Iterator[SearchGoogleAds
         client = self.client
         search_request = client.get_type("SearchGoogleAdsRequest")
         search_request.query = query
```

Comment:
without this, how do we do pagination ?

Reply:
passing a page size will cause an exception in v17. They have a default value of  10000 rows per page
The [documentation](https://developers.google.com/google-ads/api/docs/upgrade#v15-v16) also mentions it. This change is between v16 to v17